lib/app/utils/oauth2_error_messages.rb

```diff
@@ -18,6 +18,12 @@ def no_instance_for_state_error_message
         )
       end
 
+      def param_description
+        return 'No state parameter was returned by the authorization server. ' if params[:state].nil?
```

Comment:
`state` is wrapped in single quotes below so we should be consistent one way or another.
```suggestion
        return "No 'state' parameter was returned by the authorization server." if params[:state].nil?
```